public/assets/images/contentful-fragment-logo.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="239px" height="56px" viewBox="0 0 239 56" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
for svgs I think its better to remove height and width from the file itself and then just use CSS to control the width in place